policyengine_it/variables/gov/agenzia_delle_entrate/tax/income/income_tax_before_refundable_credits.py

```diff
@@ -0,0 +1,16 @@
+from policyengine_it.model_api import *
+
+
+class income_tax_before_refundable_credits(Variable):
+    value_type = float
+    entity = Person
+    definition_period = YEAR
+    unit = EUR
+    label = "Italian national income tax before refundable credits"
+    documentation = "Income tax liability (including other taxes) after non-refundable credits are used, but before refundable credits are applied"
+    adds = [
```

Comment:
this should be capped at 0 if non refundable

Reply:
formula:
    amount = household(....
    non_ref_c = household(...
return max_(amount - non_ref_c, 0)